lib/utilities/inspec_util.rb

```diff
@@ -145,18 +145,16 @@ def self.to_dotted_hash(hash, recursive_key = '')
 
     def self.control_status(control)
       status_list = control[:status].uniq
-      if status_list.include?('error')
-        result = 'Profile_Error'
-      elsif control[:impact].to_f.zero?
+      if control[:impact].to_f.zero?
         result = 'Not_Applicable'
       elsif status_list.include?('failed')
         result = 'Open'
       elsif status_list.include?('passed')
         result = 'NotAFinding'
-      elsif status_list.include?('skipped')
+      elsif status_list.include?('skipped') || status_list.include?('error')
```

Comment:
Couldn't this elsif be removed since everything else is Not_Reviewed, or did you want it to be explicit?

Reply:
I was hoping to leave it explicit so if someone was wondering what the intended status are versus the catch all it would be known instead of us forgetting.

Reply:
I really think I would prefer a comment in this case for brevity. Also the `result` can be removed from this whole function and the result of the `if else` statement would be implicitly returned (would lower ABC).